Components/ProductStream/ProductStreamService.php

```diff
@@ -191,6 +191,11 @@ public function getList($start = null, $limit = null)
         );
     }
 
+    public function isStreamExported($streamId)
+    {
+        return (bool) $this->streamAttrRepository->findOneBy(array('streamId' => (int) $streamId));
```

Comment:
Please check if the status is export. If it's error or delete, stream is not exported.